.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+name: Semantic Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  semantic-release:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
Do we need this check? According to [Github Actions docs](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/skipping-workflow-runs) this is built-in feature... :thinking: